Tests/LibWeb/Text/input/Worker/Worker-postMessage-transfer.html

```diff
@@ -24,6 +24,8 @@
         const message = encoder.encode("Hello, world");
         const myBuf = message.buffer;
 
+        println('Buffer length before transfer: ' + myBuf.byteLength);
```

Comment:
Is this left over from debugging? If not, you'll also need to update the expectation file.

Reply:
I've pushed updated `Worker-postMessage-transfer.txt` file, now it is in sync with the tests. Note that this test is ignored in `TestConfig.ini`.

I resolve this conversations as per contribution guidance.